scripts/startup-script.sh

```diff
@@ -48,3 +50,7 @@ resources_prefix: ''
 EOF
 
 cfy bootstrap -p $BLUEPRINT -i $INPUTS --install-plugins
```

Comment:
A side note: the `cfy` command **can** run as user `root` (because why not). This method of installation is also supported.

In this particular case it's more a matter of code elegance. This script was meant to be utilitarian, not a part of the system. Therefore I wouldn't touch **right now** something that works and has a high probability of being removed. ;) I just wanted to save a note for future pull requests.